docs/source/getting_started_guide/introduction.rst

```diff
@@ -0,0 +1,16 @@
+++++++++++++++++++++++++
+Introduction to receptor
+++++++++++++++++++++++++
+
+Receptor is an overlay network.
+It eases the work distribution across a large and dispersed collection
+of workers
+
+Receptor nodes establish peer-to-peer connections with each other via
+existing networks
+
+Once connected, the receptor mesh provides:
+
+* Datagram (UDP-like) and stream (TCP-like) capabilities to applications
+* Robust unit-of-work handling
+* Resiliency against transient network failures.
```

Comment:
```suggestion
* Resiliency against transient network failures
```

Reply:
Total nit, but if you have a bullet list like this, they should either all end in a period, or none of them have a period.